README.md

```diff
@@ -141,33 +141,46 @@ export ROS_SECURITY_ENABLE=true
 
 Now try starting another teleop node with security disabled and check that only the secure teleop node can drive the robot.
 
-# Security Workshop Sandbox Node Demo
+## Sandboxed Nodes Demo
 
 The [ROSCon 2019](https://ros-swg.github.io/ROSCon19_Security_Workshop/) [ROS2 Security Workshop](https://ros-swg.github.io/ROSCon19_Security_Workshop/)
 will present the opportunity to run the Turtlebot3 demo using the [launch-ros-sandbox](https://github.com/aws-robotics/launch-ros-sandbox)
-package. Specifically, the demo uses this package to launch the Turtlebot3 navigation nodes in a docker container. See the 
+package. Specifically, the demo uses this package to launch the Turtlebot3 navigation nodes in a docker container. See the
 configs/sandbox_demo/navigation_sandbox.launch.py launch file for details.
 
-## Running the Security Workshop Sandbox Node Demo
+### Running the Security Workshop Sandbox Node Demo
+
+``` bash
+rocker --x11 --nvidia rosswg/turtlebot3_demo "byobu -f configs/sandbox_demo/unsecure.conf attach"
+```
+
+Omit the `--nvidia` arg if you don't have dedicated GPU for hardware acceleration of 3D OpenGL views.
 
-Build the docker image required using the dockerfile from this repository
```

Comment:
Is it clear that the docker image will need to be built? I don't think we're hosting the new image, so doesn't it have to be generated locally?

Reply:
No - if you look at the instruction there is absolutely no difference between what you would create locally and what we would be building upstream. The instructions provide no special flags, no separate Dockerfile. It's exactly the same image that we're hosting. We should make sure the Dockerhub repo is set up to autobuild the image from master here - it's really easy to do.

Reply:
Does the hosted docker image contain the launch-ros-sandbox dependency? Notice the https://github.com/ros-swg/turtlebot3_demo/blob/master/.docker/overlay.repos: it now contains that dependency (my last PR: https://github.com/ros-swg/turtlebot3_demo/pull/14).

Reply:
@ruffsl is the https://hub.docker.com/r/rosswg/turtlebot3_demo/tags image set to auto-build `latest` from `master` here? If not, can we do that?

Reply:
@dabonnie it will once we update the image on hub

Reply:
Let's keep the `roscon19` tag as an image we manually curate and check for the tutorial. However, I'll enabled auto build for the master branch to build to the implicit `latest` tag, so pulling `rosswg/turtlebot3_demo` should pull an image derived from the latest commit.

I've having some docker cloud linking issues, so I'll resolve that later today.

Reply:
@ruffsl any luck with linking dockerhub? happy to help if needed